v2/cli/init_templates.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+)
+
+// Template project.yaml file for Brigade projects
+// nolint: lll
+var projectTemplate = []byte(`# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+  id: {{ .ProjectID }}
+description: My new Brigade project
+spec:
+  ## Subscribe below to any events that should trigger your script.
+  ## The commented example depicts a subscription to "exec"
+  ## events originating from the Brigade CLI. 
+  # eventSubscriptions:
+  # - source: brigade.sh/cli
+  #   types:
+  #   - exec
+  workerTemplate:
+    # logLevel: DEBUG
+    {{- if eq .GitCloneURL "" }}
+    defaultConfigFiles:
+      brigade.{{ .Language }}: |
+        {{ nindent 8 .Script }}
+    {{- else }}
+    git:
+      cloneURL: {{ .GitCloneURL }}
+    {{- end }}
+`)
+
+// nolint: lll
+var typeScriptTemplate = []byte(`import { events, Job } from "@brigadecore/brigadier"
+
+events.on("brigade.sh/cli", "exec", async event => {
```

Comment:
Another case where I think a comment could be instructive.  Let's try something like:

```
// Use events.on() to define how your script responds to different events. The examples below depicts
// handling of "exec" events originating from the Brigade CLI.
```

The handler itself can also be commented.

The JS template will require the same treatment.

Reply:
I added a few more comments in the handler itself. Do these make sense and does the wording look okay?

```
// Use events.on() to define how your script responds to different events. 
// The examples below depicts handling of "exec" events originating from
// the Brigade CLI.
events.on("brigade.sh/cli", "exec", async event => {
  // Use the Job() constructor to define a new job for the event.
  let job = new Job("hello", "debian:latest", event)

  // Define the commands and arguments for the job.
  job.primaryContainer.command = ["echo"]
  job.primaryContainer.arguments = ["Hello, World!"]

  // Use job.run() to run the job you have just defined.
  await job.run()
})
```

Reply:
Oh wait, do you mean that I should add comments to the handler, or comment it out?

Reply:
I'm assuming that I'm supposed to comment it out, since the exec event in `project.yaml` is commented out. I also only used one set of `//` for the comment, as I'm not sure if that's the standard for script files in the same way as yaml files

Reply:
Ya... comment _out_ the handler.